.github/ISSUE_TEMPLATE/Bug_issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Report a bug
+about: Report an issue that you have encountered.
+title: ''
+labels: 'Type: bug'
+---
+
+#### Summary
+
+<Provide information about the problem.>
+
+#### Steps to reproduce
+
+<Provide instructions for reproducing this issue.>
```

Comment:
We should probably add something here asking what version of CIL they are using? Maybe this links with @gfardell's work on printing installation information

Reply:
I modified it slightly to "Provide CIL version and instructions for reproducing this issue."